### ISSUE TITLE
converting a torch tensor to numpy array

### ISSUE THREAD

**Issue by litaPhsinaM** [litaPhsinaM/Object-Detection-on-VR_Manish-Patil#7]

**Comment by litaPhsinaM** [comment]
1. torch.Tensor().numpy()
2. torch.Tensor().cpu().data.numpy()
3. torch.Tensor().cpu().detach().numpy()
